examples/00_quick_start/dkn_MIND_dataset.ipynb

```diff
@@ -30,13 +30,13 @@
     "- training / validation / test files: each line in these files represents one instance. Impressionid is used to evaluate performance within an impression session, so it is only used when evaluating, you can set it to 0 for training data. The format is : <br> \n",
     "`[label] [userid] [CandidateNews]%[impressionid] `<br> \n",
     "e.g., `1 train_U1 N1%0` <br> \n",
-    "- user history file: each line in this file represents a users' click history, the format is : <br> \n",
+    "- user history file: each line in this file represents a users' click history. You need to set his_size parameter in config file, which is the max number of user's click history we use. We will automatically keep the last his_size number of user click history, if user's click history is more than his_size, and we will automatically padding 0 if user's click history less than his_size. the format is : <br> \n",
```

Comment:
Maybe calling this variable `history_size` would be more informative.

Reply:
yes, I changed it to history_size